### ISSUE TITLE
Late enchantment registration

### ISSUE THREAD

**Issue by LemADEC** [TeamMetallurgy/Aquaculture#52]
As of 1.12.2-1.6.8, the mod appears to register enchantment later than expected according to Apotheosis-1.12.2-1.10.2 logs:
```
[19:02:43] [Server thread/ERROR] [Apotheosis : Enchantment]: Had to late load enchantment info for aquaculture:appealing, this is a bug in the mod aquaculture as they are registering late!
[19:02:43] [Server thread/ERROR] [Apotheosis : Enchantment]: Had to late load enchantment info for aquaculture:magnetic, this is a bug in the mod aquaculture as they are registering late!
[19:02:43] [Server thread/ERROR] [Apotheosis : Enchantment]: Had to late load enchantment info for aquaculture:long_cast, this is a bug in the mod aquaculture as they are registering late!
[19:02:43] [Server thread/ERROR] [Apotheosis : Enchantment]: Had to late load enchantment info for aquaculture:short_cast, this is a bug in the mod aquaculture as they are registering late!
[19:02:43] [Server thread/ERROR] [Apotheosis : Enchantment]: Had to late load enchantment info for aquaculture:double_hook, this is a bug in the mod aquaculture as they are registering late!
[19:02:43] [Server thread/ERROR] [Apotheosis : Enchantment]: Had to late load enchantment info for aquaculture:barbed_hook, this is a bug in the mod aquaculture as they are registering late!
[19:02:43] [Server thread/ERROR] [Apotheosis : Enchantment]: Had to late load enchantment info for aquaculture:heavy_line, this is a bug in the mod aquaculture as they are registering late!
```

**Comment by LemADEC** [comment]
According to https://github.com/TeamMetallurgy/Aquaculture/blob/1.12.2/src/main/java/com/teammetallurgy/aquaculture/Aquaculture.java#L51 this registration is done during the mod Init while it should be done through the registration events.

**Comment by GirafiStudios** [comment]
It was not possible to register enchants in the RegistryEvent at the time it was coded.
Since this is not actually breaking anything, it's not going to get fixed for 1.12.2
